backend/src/controllers/course.js

```diff
@@ -2,27 +2,31 @@ const db = require('../db');
 
 const getCourses = async (req, res) => {
   const {
-    query: { classId },
+    query: { classId, studentId },
   } = req;
 
-  if (!classId) {
-    return res
-      .status(400)
-      .send({ message: 'Please provide one of the following: classId' });
+  if (!classId && !studentId) {
+    return res.status(400).send({
+      message: 'Please provide one of the following: classId, studentId',
```

Comment:
Deze error insinueert nu dat je één van de volgende (of beiden) moet providen, terwijl de if-statement alleen uitgevoerd wordt bij beiden, misschien net iets anders formuleren of er in de if-statement een `||` zetten ipv `&&`?